Source/JavaScriptCore/jit/GPRInfo.h

```diff
@@ -593,7 +593,7 @@ class GPRInfo {
     static constexpr GPRReg wasmScratchGPR0 = regT5;
     static constexpr GPRReg wasmScratchGPR1 = regT6;
     static constexpr GPRReg wasmContextInstancePointer = regCS0;
-    static constexpr GPRReg wasmBaseMemoryPointer = InvalidGPRReg;
+    static constexpr GPRReg wasmBaseMemoryPointer = ARMRegisters::r9;
```

Comment:
This is not callee-save register, right? Our strong assumption is that `wasmBaseMemoryPointer` is callee-save register.

Reply:
Could you hint as to where that assumption stems from? If there are other paths I should adjust to make sure the value is preserved, I'd like to look into them as both `regCS0 ` and `regCS1` are already spoken for (and they're all we have on ARMv7).

Reply:
For example, we are not restoring these values when calling an operation function since we are assuming they are callee-saves.

Reply:
Oh, so you mean it's not callee-save for native calls? It is on Linux but not on Darwin. Would you be OK with a version of the patch that only pins `wasmBaseMemoryPointer` on Linux?

Reply:
That sounds good.